lib/bolognese/metadata_utils.rb

```diff
@@ -159,7 +159,7 @@ def citeproc_hsh
         "title" => parse_attributes(titles, content: "title", first: true),
         "URL" => url,
         "copyright" => Array.wrap(rights_list).map { |l| l["rights"] }.first,
-        "version" => version,
+        "version" => version_info
```

Comment:
I guess this has nothing to do with `rightsidentifier` and its related to another issue. Could you help me to understand, what is this about? I thought all the version swapping was resolved.

Reply:
No, I had to reverse these changes, as we reverted the database migration. This is here as I synced the pull request with master.